### PR TITLE
add "none" to sameSite possible values

### PR DESCRIPTION
**Pull request by meddjelaili**
_Issue #4958
pull request #5726 

_Description of changes: added none to the accepted values, the none value is possible on js-cookie it's just not documented, [see this comment on js-cookie issues](https://github.com/js-cookie/js-cookie/issues/652#issuecomment-667419436).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by meddjelaili** [comment]
@jordanranz can you please check this pull request

File: packages/amazon-cognito-identity-js/src/CookieStorage.js [hunk 1]
```diff
@@ -33,11 +33,16 @@ export default class CookieStorage {
 			this.secure = true;
 		}
 		if (Object.prototype.hasOwnProperty.call(data, 'sameSite')) {
-			if (data.sameSite !== 'strict' && data.sameSite !== 'lax') {
+			if (!['strict','lax','none'].includes(data.sameSite)) {
 				throw new Error(
 					'The sameSite value of cookieStorage must be "lax" or "strict".'
```
Comment (amhinson) [inline packages/amazon-cognito-identity-js/src/CookieStorage.js:38]:
```suggestion
					'The sameSite value of cookieStorage must be "lax", "strict" or "none".'
```

**Comment by meddjelaili** [comment]
> Thanks for the PR @meddjelaili 🙏 
> 
> Could you also add 'none' as a value to the TS types?
> https://github.com/aws-amplify/amplify-js/blob/4e25f923723441c6fb705584fcc3349214806558/packages/amazon-cognito-identity-js/index.d.ts#L371

yes sure

**Comment by meddjelaili** [comment]
> Thanks again @meddjelaili! LGTM 👍

Thanks for you guys for this fantastic project

**Pull request merged**